eslint/cactbot-output-strings.js

```diff
@@ -19,10 +19,13 @@ const ruleModule = {
     messages: {
       noOutputStrings: 'no outputStrings in trigger',
       notFoundProperty: 'no \'{{prop}}\' in \'{{outputParam}}\'',
+      notFoundTemplate: '`output.{{outputParam}}(data)` doesn\'t take property \'{{prop}}\' on `data`',
+      tooMuchParams: 'function `output.{{call}}()` take only {{num}} params',
```

Comment:
Is it possible to write a "missingTemplate", where it says which template parameters are missing?

Reply:
`tooMuchParams` means `output.call()` with template or `output.call({...})` without template. 

I'll add a error for template in outputString is not fulfilled